test/unit/utils/tsv-utils.spec.ts

```diff
@@ -0,0 +1,29 @@
+import chai from 'chai';
+import 'deep-equal-in-any-order';
+import { TsvUtils } from '../../../src/utils';
+
+chai.use(require('deep-equal-in-any-order'));
+
+describe('tsv-utils', () => {
+  describe('tsv-parser', () => {
+    it.only('should remove qouble quotes for excel compatibility', () => {
+      // some example values after exported out of excel
+      const content = ` \
```

Comment:
I asked Kim to try out these values from rxdb:
```
Methylcellulose "450" 500 mg oral tablet
transparent dressing 4 3/8" X 5" TOPICAL BANDAGE
Health Care America Insulin Syringe 29gx1/2"
```

 and took the exported excel value